docs/consensus-mechanisms-blockchain.md

```diff
@@ -89,4 +89,4 @@ This mechanism is born with a very old proposal, but different from the previous
 This mechanism was born as an Intel solution to the advantage obtained by those with more resources when looking for the fairest block validator election system, like a lottery within the network.
 
 
-If you want to learn more about the different consensus mechanisms, you can visit the following [link](https://www.allerin.com/blog/8-blockchain-consensus-mechanisms-you-should-know-about).
+If you want to learn more about the different consensus mechanisms, you can visit the following [consensus mechanisms you should know about](https://www.allerin.com/blog/8-blockchain-consensus-mechanisms-you-should-know-about).
```

Comment:
visit the following **page** [consensus mechanisms you should know about]

Reply:
Fixed up